pkg/cmd/cli/secrets/add_secret_to_obj.go

```diff
@@ -42,11 +47,36 @@ var (
 	MountType SecretType = "mount"
 )
 
+type SecretTypeList []SecretType
```

Comment:
I wasn't sure why we were strongly-typing the strings in the first place, why not just use a `StringList` from the get go?